Symposium/BasicModal.jsx

```diff
@@ -0,0 +1,162 @@
+import React, { useState } from 'react';
+import { Modal, StyleSheet, View, KeyboardAvoidingView, Keyboard, FlatList, TouchableOpacity, TouchableWithoutFeedback, ScrollView } from 'react-native';
+import { Stack, TextInput, IconButton, HStack } from "@react-native-material/core";
+import { Button as Rbutton } from "@react-native-material/core";
+
+import Icon from "@expo/vector-icons/MaterialCommunityIcons";
+import { Card, Text, Button } from 'react-native-paper';
+const BasicModal = (props) => {
+    const topicList = ["Entertainment", "Politics", "IT", "Business"]
+    const [modalVisible, setModalVisible] = useState(false);
+    return (
+        <View style={styles.centeredView}>
+
+            <Modal
+                animationType="slide"
+                transparent={true}
+                visible={modalVisible}
+                onBackdropPress={() => setModalVisible(false)}
+                onRequestClose={() => {
+                    setModalVisible(!modalVisible)
+                }}
+            >
+                <View
+                    style={styles.modalContainer}>
+                    <KeyboardAvoidingView
+                        behavior={Platform.OS === 'ios' ? 'padding' : 'height'}
+                        style={{ flex: 1 }}
+                    >
+                        <View style={styles.modalView}>
+                            <TouchableWithoutFeedback onPress={Keyboard.dismiss}>
+                                <Text style={styles.modalTitle}>Have your say!</Text>
+                            </TouchableWithoutFeedback>
+                            <Stack fill spacing={8} style={{ width: "100%", marginTop: 16 }}>
+                                <ScrollView style={{ flex: 1 }}>
+                                    <TextInput
+                                        style={{ width: "100%" }}
```

Comment:
The text for the text box gets cut off on the top. I assume just adding a little bit of margin should be fine as suggested.

```suggestion
                                        style={{ width: "100%", marginTop: 4 }}
```
![image](https://user-images.githubusercontent.com/25761885/233370361-6dae4848-e0c1-4313-86b8-b075a4613039.png)